source/functions.lua

```diff
@@ -120,19 +120,38 @@ function functions.LoadGame()
 
     local savefile
     local contents
+	local size
+	local error = false
 
     savefile = savedir .. "/" .. "landers.dat"
-    contents, _ = Nativefs.read( savefile)
-    LANDERS = Bitser.loads(contents)
+
+	if not love.filesystem.getInfo(savefile) then
+	    contents, size = nativefs.read(savefile)

```

Comment:
Since `love.filesystem.getInfo` is not searching for files inside the source directory, error always was true, at least for me.
I pushed a commit to main (4bd00a5f024c4b327ecf1d1abbb21c970217feb1) that changed `love.filesystem.getInfo` for `Nativefs.getInfo`. 

It also changes the old globals to the new format.